src/components/scenes/GettingStartedScene.tsx

```diff
@@ -0,0 +1,349 @@
+      <SkipButton swipeOffset={swipeOffset}>
+        <Space isItemEnd horizontal vertical={0.5}>
+          <TouchableOpacity onPress={handlePressSkip}>
+            <EdgeText>{s.strings.skip}</EdgeText>
+          </TouchableOpacity>
+        </Space>
+      </SkipButton>
+      <SwipeOffsetDetector swipeOffset={swipeOffset} minOffset={0} maxOffset={sections.length}>
+        <Container>
+          <HeroContainer>
+            <WelcomeHero swipeOffset={swipeOffset}>
+              <Image source={edgeLogoIcon} />
+              <WelcomeHeroTitle>{s.strings.getting_started_welcome_title}</WelcomeHeroTitle>
+              <WelcomeHeroMessage>{s.strings.getting_started_welcome_message}</WelcomeHeroMessage>
+              <WelcomeHeroPrompt>{s.strings.getting_started_welcome_prompt}</WelcomeHeroPrompt>
+            </WelcomeHero>
+            {sections.map((section, index) => {
+              return (
+                <HeroItem key={section.key} swipeOffset={swipeOffset} itemIndex={index + 1}>
+                  <HeroImageContainer>
+                    <HeroImage source={section.image} />
+                  </HeroImageContainer>
+                </HeroItem>
+              )
+            })}
+          </HeroContainer>
+          <Pagination>
+            {Array.from({ length: paginationCount }).map((_, index) => (
+              <Pressable key={index} onPress={() => handlePressIndicator(index)}>
+                <PageIndicator swipeOffset={swipeOffset} itemIndex={index} />
+              </Pressable>
+            ))}
+          </Pagination>
+          <SectionCoverAnimated swipeOffset={swipeOffset}>
+            <Sections swipeOffset={swipeOffset}>
+              {sections.map((section, index) => {
+                return (
+                  <Section key={section.key} swipeOffset={swipeOffset} itemIndex={index + 1}>
+                    <ScrollView>
+                      <SectionTitle numberOfLines={2}>{parseMarkedText(section.title)}</SectionTitle>
+                      <SectionParagraph numberOfLines={undefined}>{section.message}</SectionParagraph>
+                      {section.footnote == null ? null : <Footnote numberOfLines={undefined}>{s.strings.getting_started_slide_1_footnote}</Footnote>}
+                    </ScrollView>
+                  </Section>
+                )
+              })}
+            </Sections>
+            <Space horizontal={2}>
+              <MainButton onPress={handlePressSignUp} label={s.strings.create_wallet_create_account} />
+              <MainButton type="escape" onPress={handlePressSignIn} label={s.strings.getting_started_button_sign_in} />
+            </Space>
+          </SectionCoverAnimated>
+        </Container>
+      </SwipeOffsetDetector>
+    </SceneWrapper>
+  )
+}
+
+// -----------------------------------------------------------------------------
+// Local Components
+// -----------------------------------------------------------------------------
+
+const Container = styled(View)(_ => ({
```

Comment:
Instead of `_ ->`, would `() -> ` be better? It makes it clear that we don't care about the props, as opposed to wondering if the `_` might get used somewhere (even though convention says it shouldn't).